packages/model/addon/-private/system/many-array.js

```diff
@@ -72,18 +72,8 @@ export default EmberObject.extend(MutableArray, DeprecatedEvented, {
     */
     this.isLoaded = this.isLoaded || false;
 
-    // async has-many rendering tests
     this._length = 0;
 
-    /**
-    Used for async `hasMany` arrays
-    to keep track of when they will resolve.
-
-    @property {Ember.RSVP.Promise} promise
-    @private
-    */
-    this.promise = null;
```

Comment:
Was this effectively unused?

Reply:
Yeah. It hasn't been used in many years, unclear if it was ever used at all.